sdk/appconfiguration/azure-appconfiguration/setup.py

```diff
@@ -1,24 +1,22 @@
-#!/usr/bin/env python
-
-# -------------------------------------------------------------------------
+# coding=utf-8
+# --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
+# coding: utf-8
 
+import os
 import re
-import os.path
-from io import open
-from setuptools import find_packages, setup
+from setuptools import setup, find_packages
+
 
-# Change the PACKAGE_NAME only to change folder and different name
 PACKAGE_NAME = "azure-appconfiguration"
-PACKAGE_PPRINT_NAME = "App Configuration Data"
+PACKAGE_PPRINT_NAME = "Azure Appconfiguration"
```

Comment:
?

Reply:
> ?
It's auto generated from tsp, should we use the same print name as before?

Reply:
Updated